iceoryx_platform/linux/include/iceoryx_platform/unistd.hpp

```diff
@@ -19,7 +19,18 @@
 
 #include <unistd.h>
 
+#define IOX_SEEK_SET SEEK_SET
+
+using iox_off_t = off_t;
+using iox_ssize_t = ssize_t;
```

Comment:
In `iox_read` and `iox_write`. In the end in every forward function which uses `ssize_t`. I created here an extra internal type alias so that it does not interfere with platform specific ones. Sometimes a platform introduces the same name for a variable but it is used completely different.